jax/_src/numpy/ufuncs.py

```diff
@@ -126,17 +126,83 @@ def positive(x: ArrayLike, /) -> Array:
 def sign(x: ArrayLike, /) -> Array:
   return lax.sign(*promote_args('sign', x))
 
-@implements(np.floor, module='numpy')
+
 @partial(jit, inline=True)
 def floor(x: ArrayLike, /) -> Array:
+  """Round input to the nearest integer downwards.
+
+  JAX implementation of :func:`numpy.floor`.
+
+  Args:
+    x: input array or scalar.
+
+  Returns:
+    An array with same shape and dtype as ``x`` containing the values rounded to
+    the nearest integer that is less than the value itself.
+
+  Note:
+    ``jnp.floor`` does not support ``complex`` dtypes.
```

Comment:
Rather than a separate note, I'd just put this in the description of ``x``. Same in `ceil` below.

Reply:
Done